storage/data-plane/src/test/java/com/microsoft/azure/storage/blob/RequestRetryTestFactory.java

```diff
@@ -339,8 +339,9 @@ public HttpHeaders headers() {
                         default:
                             throw new IllegalArgumentException("Retries continued on non retryable error.");
                     }
+                default:
+                    throw new IllegalArgumentException("Invalid retry test scenario.");
             }
-            return Single.error(new IllegalArgumentException("Invalid scenario"));
```

Comment:
I removed this line because after adding the default case, this line has never reached. Let me know if there is concern here.